server/builtin/info.go

```diff
@@ -40,5 +43,10 @@ func (a *builtinApp) info(_ *incoming.Request, creq apps.CallRequest) apps.CallR
 		},
 		TemplateData: conf.InfoTemplateData(),
 	}) + "\n"
+
+	if conf.DeveloperMode && conf.AWSAccessKey != "" {
+		out += fmt.Sprintf("AWS config: region: `%s`, bucket: `%s`, access: `%s`, secret: `%s`",
```

Comment:
Should this be localized?

Reply:
I'd say, not, but 0/5. this is debugging only.

Reply:
Given that the info command is available even if Debug mode is disabled and the rest of the command is localized, I'm 2/5 that this addition should also be localized.